src/views/AuthenticationWrapper/index.tsx

```diff
@@ -15,6 +15,17 @@ type ParentComponentProps = {
 const AuthenticationWrapper = ({ children }: ParentComponentProps) => {
   const history = useHistory();
 
+  const authClient = new OktaAuth({
+    issuer: process.env.REACT_APP_OKTA_ISSUER,
+    clientId: process.env.REACT_APP_OKTA_CLIENT_ID,
+    redirectUri: process.env.REACT_APP_OKTA_REDIRECT_URI,
+    responseType: ['code'],
+    tokenManager: {
+      autoRenew: false
+    },
+    pkce: true
```

Comment:
It sounds like `pkce: true` is the default, so we might not need to set it anymore.